Description: Add build files for react window
 Embedded copy is from npmjs.com so it is missing some files required to build.
Forwarded: not-needed

--- /dev/null
+++ b/packages/react-devtools-shared/src/node_modules/react-window/flow-template
@@ -0,0 +1,3 @@
+// @flow
+
+export * from '../src';
--- /dev/null
+++ b/packages/react-devtools-shared/src/node_modules/react-window/rollup.config.js
@@ -0,0 +1,96 @@
+import path from 'path';
+import babel from 'rollup-plugin-babel';
+import replace from 'rollup-plugin-replace';
+import { terser } from 'rollup-plugin-terser';
+import commonjs from 'rollup-plugin-commonjs';
+import nodeResolve from 'rollup-plugin-node-resolve';
+
+import pkg from './package.json';
+
+const input = './src/index.js';
+
+const external = id => !id.startsWith('.') && !path.isAbsolute(id);
+
+const babelConfigEsModules = babel({
+  runtimeHelpers: true,
+  plugins: [['@babel/transform-runtime', { useESModules: true }]],
+  sourceMaps: true,
+});
+
+const umdGlobals = {
+  react: 'React',
+};
+
+export default [
+  {
+    input,
+    output: {
+      file: pkg.main,
+      format: 'cjs',
+      sourcemap: true,
+    },
+    external,
+    plugins: [
+      babel({
+        runtimeHelpers: true,
+        plugins: ['@babel/transform-runtime'],
+        sourceMaps: true,
+      }),
+      nodeResolve(),
+      commonjs(),
+    ],
+  },
+
+  {
+    input,
+    output: {
+      file: pkg.module,
+      format: 'esm',
+      sourcemap: true,
+    },
+    external,
+    plugins: [babelConfigEsModules, nodeResolve(), commonjs()],
+  },
+
+  {
+    input,
+    output: {
+      file: 'dist/index-dev.umd.js',
+      format: 'umd',
+      sourcemap: true,
+      name: 'ReactWindow',
+      globals: umdGlobals,
+    },
+    external: Object.keys(umdGlobals),
+    plugins: [
+      babelConfigEsModules,
+      nodeResolve(),
+      commonjs(),
+      replace({
+        'process.env.NODE_ENV': JSON.stringify('development'),
+      }),
+      terser(),
+    ],
+  },
+
+  {
+    input,
+    output: {
+      file: 'dist/index-prod.umd.js',
+      format: 'umd',
+      sourcemap: true,
+      name: 'ReactWindow',
+      globals: umdGlobals,
+    },
+    external: Object.keys(umdGlobals),
+    plugins: [
+      babelConfigEsModules,
+      nodeResolve(),
+      commonjs(),
+      replace({
+        'process.env.NODE_ENV': JSON.stringify('production'),
+      }),
+      terser(),
+    ],
+  },
+];
--- /dev/null
+++ b/packages/react-devtools-shared/src/node_modules/react-window/babel.config.js
@@ -0,0 +1,7 @@
+module.exports = {
+  presets: [['@babel/env', { loose: true }], '@babel/flow'],
+  plugins: [
+    ['@babel/proposal-class-properties', { loose: true }],
+    'annotate-pure-calls',
+  ],
+};
